src/mbgl/shader/circle_uniforms.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <mbgl/shader/uniforms.hpp>
+
+namespace mbgl {
+
+MBGL_DEFINE_UNIFORM_SCALAR(float, u_radius);
```

Comment:
Okay; but it still feels weird to have them in the main `mbgl` namespace so we have symbols like `mbgl::u_radius`. Can we at least move them to another namespace?